lib/main_transactional.pm

```diff
@@ -0,0 +1,283 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader for Transactional product family:
```

Comment:
A bit misleading because it doesn't cover transactional server...

Calling it `lib/main_micro.pm` is a bit misleading as well though.

Reply:
```suggestion
# Summary: module loader for various transactional products:
```

Perhaps?

Reply:
Yes, both can be misleading.. `main_micro_alp.pm` ?